docs/development/roadmap.rst

```diff
@@ -0,0 +1,17 @@
+.. _roadmap:
+
+#######
+Roadmap
+#######
+
+This page serves as a reference to the overall goals of NiBetaSeries development.
+
+Goals
+-----
+
+1. Flexibly generate beta series maps for the user.
+2. Provide analyses of beta series maps for the user.
+3. Keep code test coverage at 70%.
```

Comment:
It would be nice to explain the 70% criterium for new users.

Reply:
I think the [testivus](https://www.artima.com/weblogs/viewpost.jsp?thread=204677) answer is pretty good/funny.

Reply:
That's amazing!